.bones/tasks/marky-6n3.md

```
@@ -0,0 +1,39 @@
+type: task
+priority: 3
+owner: sethyanow@users.noreply.github.com
+depends_on: [marky-ccv]
+parent: marky-8s3.7
+---
+
+
+
+## Design
+
+## Goal
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)